File: src/ChangelogApi/Models/ChangeLog.cs
using System;

namespace ChangelogApi.Models
{
    public class ChangeLog
    {
        public int Id { get; set; }
        public string Version { get; set; }
        public string Message { get; set; }
        public string Username { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }
    }
}

File: src/Changelog/scripts/changelog.ts
export class ChangeLog {
    id: number;
    version: string;
    message: string;
    username: string;
    createdOn: Date;
    updatedOn: Date;

    constructor(version, message, username) {
        this.version = version;
        this.message = message;
        this.username = username;

    }
}